test/parallel/test-fs-open-mode-mask.js

```diff
@@ -7,13 +7,8 @@ const assert = require('assert');
 const path = require('path');
 const fs = require('fs');
 
-let mode;
+const mode = (common.isWindows) ? 0o444 : 0o644;
```

Comment:
Nit: Parentheses unnecessary.

Reply:
I know that they are not necessary for simple case (like this one). ESLint didn't remove them (maybe I should set up ESLint before commit in Webstorm)